site/docs/about/new.mdx

```diff
@@ -13,7 +13,12 @@ import Link from "../../src/components/Link";
     Here you will find summarized patch notes of major releases of HDS. For full patch notes for each release, please refer to the <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases" external>GitHub releases</Link>.
 </LargeParagraph>
 
-<p><strong>0.22.0 (beta)</strong> - <em>3.3.2021</em> - <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/tag/v0.22.0" external>Release notes</Link></p>
+<p><strong>0.23.0 (beta)</strong> - <em>12.3.2021</em> - <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/tag/v0.22.0" external>Release notes</Link></p>
```

Comment:
Should the link point to version v0.23.0?

Reply:
Fixed